keras_core/backend/torch/math.py

```diff
@@ -81,3 +81,78 @@ def qr(x, mode="reduced"):
         )
     x = convert_to_tensor(x)
     return torch.linalg.qr(x, mode=mode)
+
+
+def _get_complex_tensor_from_tuple(a):
+    if not isinstance(a, (tuple, list)) or len(a) != 2:
+        raise ValueError(
+            "Input `a` should be a tuple of two tensors - real and imaginary."
+            f"Received: a={a}"
+        )
+    # `convert_to_tensor` does not support passing complex tensors. We separate
+    # the input out into real and imaginary and convert them separately.
+    real, imag = a
+    real = convert_to_tensor(real)
+    imag = convert_to_tensor(imag)
+    # Check shape.
+    if real.shape != imag.shape:
+        raise ValueError(
+            "Input `a` should be a tuple of two tensors - real and imaginary."
+            "Both real and imaginary should have the same shape. "
+            f"Received: real.shape = {real.shape}, imag.shape = {imag.shape}"
+        )
+    # Ensure dtype is float.
+    if not torch.is_floating_point(real) or not torch.is_floating_point(imag):
+        raise ValueError(
+            "At least one tensor in input `a` is not of type float."
+            f"Received: a={a}."
+        )
+
+    complex_input = torch.complex(real, imag)
+    return complex_input
+
+
+def fft(a, n=None, axis=-1, norm=None):
+    # TF does not support the following args. Hence, we are not supporting
+    # them right now.
+    if n is not None:
```

Comment:
My general take is that if we do not support these arguments then they should not be in the signature. The values used should be documented in the public-facing docstrings (for the `keras_core.ops` functions).